wiki/Community/Discord_servers/osu!_Medal_Hunters/de.md

```diff
@@ -32,10 +32,10 @@ Viele Medaillenjäger arbeiten zusammen, um Lösungen für jede neue Hush-Hush-M
 
 | Rolle | Benutzer (Discord-ID) |
 | :-- | :-- |
-| Server-Eigentümer | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151) |
-| Server-Admin | ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) (Hubz#6283, Tanza#8733), ::{ flag=DE }:: [mulraf](https://osu.ppy.sh/users/1309242) (mulraf#1087) |
-| Community-Manager & Server-Admin | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151), ::{ flag=GB }:: [chromb](https://osu.ppy.sh/users/10238680) (chromb#4528), ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) (Remyria#0001) |
-| Server-Moderator | ::{ flag=US }:: [bentokage](https://osu.ppy.sh/users/13175102) (bento#6969) |
+| Server-Eigentümer | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (@megamix_craft) |
+| Server-Admin | ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) (@tanza3d, @tanza.n), ::{ flag=DE }:: [mulraf](https://osu.ppy.sh/users/1309242) (@mulraf) |
```

Comment:
could you exclude the second handle here for me? it's my personal account, thanks (likewise on the other languages)

Reply:
I thought the same